.travis.yml

```diff
@@ -0,0 +1,61 @@
+# <~> Travis integration file.
+# Seattle supports Python 2.5 through 2.7, per https://seattle.poly.edu/wiki/ProgrammersPage
+# Travis provides OS X and Linux (think generally Debian-based) OS instances.
+# AppVeyor provides Windows
+
+matrix:
```

Comment:
Let's add a comment subsection mentioning that the following block is for Linux VMs. (See below for OS X, where this comment exists).